debug_toolbar/apps.py

```diff
@@ -206,3 +207,23 @@ def js_mimetype_check(app_configs, **kwargs):
             )
         ]
     return []
+
+
+@register()
+def debug_toolbar_namespace_check(app_configs, **kwargs):
+    """
+    Check that the Django Debug Toolbar is registered as a namespace.
+    """
+    if (
+        settings.DEBUG is False
+        and dt_settings.get_config()["DEBUG_TOOLBAR_CONFIG"] == "test" in sys.argv
```

Comment:
```suggestion
        and dt_settings.get_config()["DEBUG_TOOLBAR_CONFIG"]
```
The check on `sys.argv` is handled in the settings and is unnecessary to do here.

Reply:
Since its a condition I think it still needs to be there. I tried to remove it and got the djdt error.